### ISSUE TITLE
HeroChat to IRC not working in some cases.

### ISSUE THREAD

**Issue by ryan-clancy** [cnaude/PurpleIRC#4]
I've noticed that the HeroChat to IRC relay only works in some cases. 

At first, I had the channel "#test-game" as the first channel in the list in my bot config - the relay worked properly (irc -> herochat and herochat -> irc) while other channels didn't, they only had irc -> herochat. When I switched a new channel to the first in the list, "#test-help" for example, and moved "#test-game" to a position further down, "#test-help" worked properly (irc -> herochat relay and herochat -> irc relay) but "#test-game" now only has irc -> herochat relay.

tl;dr It appears that only the first channel in the list has both irc to herochat and herochat to irc relay working, but all the other only have irc to herochat and NOT herochat to irc.
